examples/suit_update/README.md

```diff
@@ -37,7 +37,7 @@ Table of contents:
 
 - Install aiocoap from the source
 
-      $ pip3 install --user --upgrade "git+https://github.com/chrysn/aiocoap#egg=aiocoap[all]"
+      $ pip3 install --user aiocoap==0.4.1
```

Comment:
Wouldn't pip automatically install the latest version?

Reply:
It does, my issue was that in my current setup I was using 0.4b3, and with `pip install aiocoap>=0.4.1` it did not update the version to `0.4.1`, while from a fresh environment `pip install aiocoap` does install the desired version (0.4.1). But in the purpose of this example in both versions the fileserver is available. So ACK

Reply:
I changed to `pip install aiocoap>=0.4.1`.